page/ajax/working-with-jsonp.md

```diff
@@ -31,4 +31,8 @@ $.ajax({
 });
 ```
 
+If you'd like to access community tables you have to add `env: "http://datatables.org/alltables.env"` to the data object otherwise you would get an error message with `No definition found for Table`.
```

Comment:
I think YQL is pretty complicated for beginners but the tip for the community tables is helpful. Because then you can do requests to Google, Github etc. 
Maybe a simpler JSONP example would help. But I have not found many good json resources. I've found wikipedia (also not easy to do requests) or http://date.jsontest.com/ for timestamp request (that's easy).